src/ansys/dpf/core/field.py

```diff
@@ -620,9 +620,15 @@ def unit(self):
     def unit(self, value):
         """Change the unit for the field.
 
+        A single string is interpreted as a known physical unit with an associated homogeneity.
+
+        A tuple of two strings is interpreted as a homogeneity and a unit name.
```

Comment:
@rafacanton while reviewing the issue with version for the test, I realized we need to document this is only available for DPF 11.0 and above, as well as test for this before the server call to raise a clean Python error.